TYPO3.Flow/Classes/TYPO3/Flow/Command/DoctrineCommandController.php

```diff
@@ -178,17 +187,33 @@ public function entityStatusCommand($dumpMappingData = false)
             $this->output('You do not have any mapped Doctrine ORM entities according to the current configuration. ');
             $this->outputLine('If you have entities or mapping files you should check your mapping configuration for errors.');
         } else {
-            $this->outputLine('Found %d mapped entities:', array(count($info)));
-            foreach ($info as $entityClassName => $entityStatus) {
-                if ($entityStatus instanceof ClassMetadata) {
-                    $this->outputLine('[OK]   %s', array($entityClassName));
+            $this->outputLine('Found %d mapped entities:', [count($info)]);
+            $this->outputLine();
+            if ($entityClassName === null) {
+                foreach ($info as $entityClassName => $entityStatus) {
+                    if ($entityStatus instanceof ClassMetadata) {
+                        $this->outputLine('<success>[OK]</success>   %s', [$entityClassName]);
+                        if ($dumpMappingData) {
+                            Debugger::clearState();
+                            $this->outputLine(Debugger::renderDump($entityStatus, 0, true, true));
+                        }
+                    } else {
+                        $this->outputLine('<error>[FAIL]</error> %s', [$entityClassName]);
+                        $this->outputLine($entityStatus);
+                        $this->outputLine();
+                    }
+                }
+            } else {
+                if (array_key_exists($entityClassName, $info) && $info[$entityClassName] instanceof ClassMetadata) {
+                    $entityStatus = $info[$entityClassName];
+                    $this->outputLine('<success>[OK]</success>   %s', [$entityClassName]);
                     if ($dumpMappingData) {
                         Debugger::clearState();
                         $this->outputLine(Debugger::renderDump($entityStatus, 0, true, true));
                     }
                 } else {
-                    $this->outputLine('[FAIL] %s', array($entityClassName));
-                    $this->outputLine($entityStatus);
+                    $this->outputLine('<info>[FAIL]</info> %s', [$entityClassName]);
```

Comment:
<nitpick>If it's a "real" error we could do `<error>[FAIL]</error>` and `$this->quit(1)`</nitpick>